### ISSUE TITLE
Bulk send support

### ISSUE THREAD

**Issue by kkamkou** [kkamkou/node-gelf-pro#24]
Add a new functionality to send multiple messages at once.

**Comment by rosygupta** [comment]
hi @kkamkou ,
Currently can you confirm is chunking is done in the default message sending via UDP? I tried sending message of 25353 bytes which got trimmed and sent only the first half of the message.
Thanks

**Comment by kkamkou** [comment]
Yes, chunking is implemented from the first version of the library (UDP). Also consider is that `32766` bytes is the maximum length for a field (message might have many fields).

Please do not spam in the tickets (instead, you're welcome to open a new ticket with any question).
